.github/workflows/hooks-warnings.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  comment:
+    if: ${{ github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            warnings_map = new Map()
+            const link_regex = /\[All logs\]\((?<url>https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*))\)/g;
+            for (const match of context.payload.comment.body.matchAll(link_regex))
+            {
+              prefix = "https://c3i.jfrog.io/c3i/misc/summary.html?json="
+              if(!match.groups.url.startsWith(prefix))
+                 continue;
+              url = match.groups.url.slice(prefix.length)
+              const result = await github.request({
+                url: url,
+              });
+              for (const job of result.data)
+              {
+                if (job.build == null)
+                  continue;
+                const log = await github.request({
+                  baseUrl: "https://c3i.jfrog.io/c3i/misc/",
+                  url: job.build,
+                });
+                prefix = "[HOOK - conan-center.py] "
+                warnings = log.data.split("\n").filter(line => line.startsWith(prefix) && line.includes(" WARN: "))
+                if(warnings.length == 0)
+                  continue
+                if(!warnings_map.has(job.reference))
+                  warnings_map.set(job.reference, new Set())
+                for(const warning of warnings)
+                {
+                  msg = warning.slice(prefix.length)
+                  warnings_map.get(job.reference).add(msg)
+                  core.warning(msg)
+                }
+              }
+            }
+            if(warnings_map.size > 0)
+            {
+              comment = "<details>\n"
+              comment += "<summary>Hooks produced the following warnings</summary>\n\n"
+              for (const [ref, warnings] of warnings_map)
+              {
+                comment += "<details>\n"
+                comment += "<summary>" + ref + "</summary>\n\n```\n"
+                for(const warning of warnings)
+                  comment += warning + "\n"
+                comment += "\n```\n</details>\n"
+              }
+              comment += "</details>\n"
+              github.rest.issues.createComment({
```

Comment:
Will the token from the public forks have enough permissions to post comments? I usually assume not, but perhaps there's trick here?

Reply:
According to https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token it depends on the enterprise/organization/repository setting

Reply:
I think it should work. The action triggers on comments, so it is somehow internal to the repository, the `GITHUB_TOKEN` should have default permissions (write). Anyway, usually the best way to test it is just to run it.